velodyne_pointcloud/src/lib/rawdata.cc

```diff
@@ -196,7 +219,14 @@ inline float SQR(float val) { return val*val; }
   /** Set up for on-line operation. */
   boost::optional<velodyne_pointcloud::Calibration> RawData::setup(ros::NodeHandle private_nh)
   {
-    private_nh.param("model", config_.model, std::string("64E"));
+
+    if (!private_nh.getParam("model", config_.model))
+    {
+      config_.model =  std::string("64E");
+      ROS_ERROR("No Velodyne Sensor Model specified using default %s!", config_.model.c_str());
+
+    }
+//    private_nh.param("model", config_.model, std::string("64E"));
```

Comment:
Please remove commented-out code. We can use `git` to look back at what was here if necessary.